cds/modules/previewer/templates/cds_previewer/macros/player.html

```diff
@@ -72,11 +83,29 @@
            }
        ];
       {% endif %}
+      {%- if embed_config.autoplay -%}
+      player.autoplay = true;
+      {% endif %}
+      // not working currently
+      {%- if embed_config.loop -%}
+      player.loop = true;
+      {% endif %}
+      {%- if embed_config.controlsOff -%}
+      player.controls = false;
```

Comment:
what about if you convert all this boolean to one line, for readability?
I don't have a great proposal, but what about something like this:
```
player.controls = {{ embed_config.controlsOff | default(boolean=False) l | lower }};
```
or
```
player.controls = {{ embed_config.controlsOff | default(boolean=False) l | tojson }};
```
to be tested...

Reply:
works with:
player.controls = {{ embed_config.controlsOff | default(False) | tojson }};